frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/useInitialAutoLayout.ts

```diff
@@ -25,14 +27,18 @@ export const useInitialAutoLayout = () => {
       return
     }
 
+    const tableNodesInitialized = nodes
+      .filter((node) => node.type === 'table')
+      .some((node) => node.measured)
+
```

Comment:
Before adding the Group Node, nodesInitialized would become true only when the drawing occurred after the conversion to TableNode was completed by `convertDBStructureToNodes()`.

However, with the addition of the Group Node, nodesInitialized started to become true even before the conversion to TableNode was completed. As a result, we created our own custom `nodesInitialized`.

Reply:
I think that makes sense!